src/tachijs.ts

```diff
@@ -133,33 +132,74 @@ class TachiJSApp<C> {
 
   bindHandler(
     router: express.Router,
+    controllerMeta: ControllerMeta,
     methodMeta: HttpMethodMeta,
     handler: express.RequestHandler
   ) {
     switch (methodMeta.method) {
       case 'get':
-        router.get(methodMeta.path, ...methodMeta.middlewares, handler)
+        router.get(
+          methodMeta.path,
+          ...controllerMeta.middlewares,
```

Comment:
Replace with

```ts
...middlewares
```

and fix other methods too.

Reply:
Yeah, that makes the code look so much cleaner 👍.